### PR TITLE
Adding in hide/show API for containers

### PR DESCRIPTION
**Pull request by jonathanKingston**
@groovecoder adds in a horrible looking button to toggle the API change.

### CODE REVIEW

File: webextension/js/popup.js [hunk 1]
```diff
@@ -1,24 +1,58 @@
+const identityState = {
+};
+
 browser.contextualIdentities.query({}).then(identites=> {
   let customContainerStyles = '';
   const identitiesListElement = document.querySelector('.identities-list');
 
-  identites.forEach(identity=> {
+  identites.forEach(identity => {
```
Comment (groovecoder) [inline webextension/js/popup.js]: [eslint-config-nightmare-mode](https://github.com/jonathanKingston/eslint-config-nightmare-mode) says `Unexpected space before =>. (arrow-spacing)` :trollface:

File: webextension/js/popup.js [hunk 2]
```diff
@@ -1,24 +1,58 @@
...
     const identityRow = `
-    <tr>
+    <tr data-identity-id="${identity.cookieStoreId}" >
```
Comment (groovecoder) [inline webextension/js/popup.js]: `identity-id` is ambiguous; `identity-cookieStoreId` is better.

File: webextension/js/popup.js [hunk 3]
```diff
@@ -1,24 +1,58 @@
...
       <td>&gt;</td>
     </tr>`;
 
     identitiesListElement.innerHTML += identityRow;
+
+  });
+
+  const rows = identitiesListElement.querySelectorAll('tr');
+  rows.forEach(row => {
+    row.addEventListener('click', (e) => {
```
Comment (groovecoder) [inline webextension/js/popup.js]: `Unexpected parentheses around single function argument. (arrow-parens)` :trollface:

File: webextension/js/popup.js [hunk 4]
```diff
@@ -1,24 +1,58 @@
...
 
+
 document.querySelector('#edit-containers-link').addEventListener('click', ()=> {
   browser.runtime.sendMessage('open-containers-preferences').then(()=> {
     window.close();
   });
 });
+
+
+function hideContainer(containerId) {
```
Comment (groovecoder) [inline webextension/js/popup.js]: Move these function definitions above their usage?

**Comment by jonathanKingston** [comment]
gifrecord_2016-12-24_024257

@groovecoder looks like it works for me with one exception to the current selected tab not hiding away, we should probably address this in a follow up bug and also ensure if that is what we actually want also. I can do follow ups of pushing the tab to about:blank then using history to try and prevent the network issues Tanvi is rightly concerned about.

**Comment by groovecoder** [comment]
I added a commit to use the hide/un-hide icons and submitted a new PR:

https://github.com/mozilla/testpilot-containers/pull/24

Follow-ups:
https://github.com/mozilla/testpilot-containers/issues/25
and
https://github.com/mozilla/testpilot-containers/issues/26

**Pull request closed without merging**